src/decorator/string/IsDateString.ts

```diff
@@ -24,7 +24,7 @@ export function IsDateString(
       name: IS_DATE_STRING,
       constraints: [options],
       validator: {
-        validate: (value, args): boolean => isDateString(value),
+        validate: (value, args): boolean => isDateString(value, args.constraints[0]),
```

Comment:
This should work and reads better, no?

```suggestion
        validate: (value, args): boolean => isDateString(value, options),
```